src/main/java/mate/academy/controllers/cars/AddDriverToCarController.java

```diff
@@ -1,4 +1,4 @@
-package mate.academy.controllers;
+package mate.academy.controllers.cars;
```

Comment:
the packages should have a singular naming "controllers -> controller", cars -> cars